api/src/paths/dwc/submission/{submissionId}/normalize.test.ts

```diff
@@ -0,0 +1,199 @@
+import chai, { expect } from 'chai';
+import { describe } from 'mocha';
+import OpenAPIRequestValidator, { OpenAPIRequestValidatorArgs } from 'openapi-request-validator';
+import OpenAPIResponseValidator, { OpenAPIResponseValidatorArgs } from 'openapi-response-validator';
+import sinon from 'sinon';
+import sinonChai from 'sinon-chai';
+import * as db from '../../../../database/db';
+import { ApiGeneralError } from '../../../../errors/api-error';
+import { DarwinCoreService } from '../../../../services/dwc-service';
+import * as keycloakUtils from '../../../../utils/keycloak-utils';
+import { DWCArchive } from '../../../../utils/media/dwc/dwc-archive-file';
+import { getMockDBConnection, getRequestHandlerMocks } from '../../../../__mocks__/db';
+import * as normalize from './normalize';
+import { POST } from './normalize';
+
+chai.use(sinonChai);
+
+describe('normalize', () => {
+  describe('openApiSchema', () => {
+    describe('request validation', () => {
+      const requestValidator = new OpenAPIRequestValidator(POST.apiDoc as unknown as OpenAPIRequestValidatorArgs);
+
+      const basicRequest = {
```

Comment:
Lets not define anything at the top level like this, and rather use `getRequestHandlerMocks` in each of the `it`s.

Reply:
More repeated code, but easier to read/maintain.